cl/assets/static-global/css/override.css

```diff
@@ -1399,3 +1399,7 @@ textarea {
   max-width: 100%;
   height: auto;
 }
+
+#vlex {
+    background: black;
+}
```

Comment:
If this is the best logo we've got for them, we should just add a black background in the png file instead of carrying extra CSS around forever.

Reply:
I left it black because I wasn't sure if we wanted something more subtle so we could debate the color or if they had a different image which it looks like they do so we can drop this